### ISSUE TITLE
Error: Could not find command "readme"

### ISSUE THREAD

**Issue by zpdcodes** [rubygems/gemstash#203]
An error raised when run `gemstash` or `gemstash -h`:

    Could not find command "readme"

However the `gemstash start` works fine.

**Comment by olleolleolle** [comment]
Steps to reproduce:

Started with `gem install gemstash`:

```
$ gem install gemstash
$ gemstash --version
Gemstash version 2.0.0
$ gemstash --help
Could not find command "readme".
```

**Comment by olleolleolle** [comment]
Hm, could it be that the distributed .txt files are not found?

```ruby
        page = File.expand_path("../man/gemstash-#{command}.#{section}", __FILE__)
```

could be turned into

```ruby
        page = File.expand_path("../man/gemstash-#{command}.#{section}.txt", __FILE__)
```

**Comment by gepser** [comment]
Hi, I investigated this issue and here is what I found:

**Status**:
In the latest release (`2.0.0`) the task `doc` (`rake doc`) didn't run (or ran with errors) so the docs are missing. Gemstash works just fine but you can't see any docs from the cli.

**Possible solution**:
Make a new release running `rake doc` before actually running `rake release`. I guess you can make one task to depend on the other.

**Tests**:
I actually tested all of this on my machine and inside a docker container and it works fine, docs are not there if you don't run `rake doc` and docs are there when you actually run `rake doc`.

**Notes**:
- On my tests I got this error: `You need to install pandoc to generate documentation` so you are gonna need `pandoc` installed on the machine to do the release.
- Part of the release process is to publish the gem on rubygems and I guess I would need special permissions for that. That's why I am not making the release myself.
- Anyhow, I just wanted to document my findings before trying to solve another issue.

 I hope this helps.

**Comment by bronzdoc** [comment]
This is fixed in `Gemtash 2.1.0` thanks all!
